### PR TITLE
Add to doc about amr deployment

### PR DESCRIPTION
**Pull request by julia-pu**

### CODE REVIEW

File: scst-store/deployment-details.hbs.md [hunk 1]
```diff
@@ -6,23 +6,27 @@ This topic describes how you can deploy and configure your Kubernetes cluster fo
...
-    - deployment
-    - replicaset
-    - Pod
-- Persistent volume claim
-- External IP address (based on a deployment configuration set to use `LoadBalancer`).
-- A Kubernetes secret to allow pulling SCST - Store images from a registry.
+- Four components:
+    - metadata-store API backend 
+    - database
+    - AMR API backend (If AMR is deployed)
```
Comment (ohlookadollar) [inline scst-store/deployment-details.hbs.md]: perhaps link to how to deploy to AMR

File: scst-store/deployment-details.hbs.md [hunk 2]
```diff
@@ -35,30 +39,56 @@ The following connection methods are recommended based on Tanzu Application Plat
 
 For a production environment, VMware recommends that you install SCST - Store with ingress enabled.
 
+#### <a id='amr'></a>Deploying AMR
```
Comment (ohlookadollar) [inline scst-store/deployment-details.hbs.md:42]: Put a warning not to use AMR in production

**Review by ohlookadollar — APPROVED** [review]

File: scst-store/deployment-details.hbs.md [hunk 3]
```diff
@@ -6,23 +6,27 @@ This topic describes how you can deploy and configure your Kubernetes cluster fo
...
-    - replicaset
-    - Pod
-- Persistent volume claim
-- External IP address (based on a deployment configuration set to use `LoadBalancer`).
-- A Kubernetes secret to allow pulling SCST - Store images from a registry.
+- Four components:
+    - metadata-store API backend 
+    - database
+    - AMR API backend (If AMR is deployed, see [Deploying AMR](#amr))
+    - AMR persister (If AMR is deployed, see [Deploying AMR](#amr))
```
Comment (derwei) [inline scst-store/deployment-details.hbs.md]: Can we change "AMR persister" to "AMR CloudEvent Handler" ? That's how we're referring to it in the observer docs (and I think how we want to refer to it externally)

Comment (julia-pu) [reply scst-store/deployment-details.hbs.md]: made the change!

**Comment by julia-pu** [comment]
Closing this MR and making an MR to upstream docs-tap

**Pull request closed without merging**